Misc/NEWS.d/next/Build/2024-05-29-17-40-50.gh-issue-119729.k0xJ5U.rst

```diff
@@ -0,0 +1,3 @@
+On POSIX systems, the pkg-config (``.pc``) files now include the ABI flags.
+For example, the free-threaded build generates ``python-3.14t.pc`` and the
```

Comment:
This also changes the debug one to have a "d" as well no?

Reply:
Yeah. Do you think that's problematic or should I just update the NEWS to make that more clear?

For context, we already do that for things like the `include` directories, which look like `include/python3.14td` for the free-threaded debug builds.

Reply:
Just update the news entry because that's the only change that affects normal builds (maybe even mentioning it in the what's new).